README.rst

```diff
@@ -1,33 +1,19 @@
 Overall Status
```

Comment:
You cannot make direct edits to this file.
It is generated by a rule at https://github.com/ni/nimi-python/blob/71d6c003e4c7f2d1ac0160e446e15d9edefca0cc/build/Makefile#L146.

(This should not be confused with the rule at https://github.com/ni/nimi-python/blob/71d6c003e4c7f2d1ac0160e446e15d9edefca0cc/build/rules.mak#L146 which generates api-specific READMEs).

DOCS_DIR is /docs
STATIC_DOCS_DIR is /docs/_static

The static files are not generated.

Reply:
This is also the reason Travis-CI failed.